### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by leonwong** [civiform/civiform#1639]
**Describe the bug**
Umbrella bug capturing accessibility issues found by Google in July. Catalog of specific issues attached. Overall, these are P3 bugs that should have low overall user impact, but we may want to address if accessibility is a priority for UDP.

**Summary of the findings**
The majority of the bugs found were related to 

- Screen reader (label, focus, unable to reach controls), keyboard focus, improper headings (No H1's on page, heading hierarchy), error cases, color contrast,  Large text , high contrast, large text, page titles etc. 

  - Missing/Improper labels prevent screen reader users from using the site as they would be unable to identify the purpose of the form fields. 
  - section headings within pages allows screen reader users to jump sections within pages instead of moving onto every control on page
  - Color contrast issues would make it difficult for individuals with visual impairments from seeing certain items on the page hence, preventing them from using the site. 
  

**Platforms used for testing** 

- Desktop - web :
  - OS | Browser | Screen Reader
  - macOS Big Sur:11.4| Safari Version 14.1.1 (16611.2.7.1.4)  | VoiceOver
  - ChromeBook | Chrome Version: 90.0.4430.218 (Official Build) (64-bit) | ChromeVox
  - Window | Chrome Version 91.0.4472.77 (Official Build) (64-bit) | JAWS/NVDA
  - Window | Firefox Version: 87.0 (64-bit) | JAWS
- Mobile Web (Android) : 
  - Pixel 3 XL, Android version: 11, Android build number: RQ2A.210505.002,Chrome browser: 91.0.4472.120
  - Pixel 4 XL , android version: 11 (build number: RQ2A.210505.002) browser: chrome
- Mobile Web (iOS) :
  - Iphone mini 12 , version: 14.2.1 browser: safari
  - IPad (5th generation) Software Version: 14.4.1 browser: safari

The total number of issues identified:  68 on all 3 platforms combined. Issues reproduced on multiple platforms were merged as part of a single bug.

Date tested between:  June 16 - July 19

**Additional context**
[issues.csv](https://github.com/seattle-uat/civiform/files/7297012/issues.csv)

**Comment by jeanniefu** [comment]
Closing this bug as it is quite old. We've made a lot of a11y improvements since then, and have a more recent full audit for WCAG 2.1 standards. All bugs surfaced from the recent audit are here: https://github.com/civiform/civiform/issues?q=is%3Aopen+is%3Aissue+label%3Aa11y+%22WCAG+2.1%22+in%3Atitle
